book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Please make sure that a method for adding rentals is present. The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Like this :point_down: 
  
  ```ruby
  def add_rental(person, date)
      Rental.new(date, self, person)
  end
  ```